### DISCUSSION TITLE
Encountered error during initialization v2.24.1

### DISCUSSION THREAD

**Discussion by akandil7** [prisma/prisma#7504] [Q&A]
Everything was working properly until I updated the prisma version to the latest one 
This was the issue :
` Encountered error during initialization : .. 
error: Environment variable not found: DATABASE_URL.`

Everything is working  (validate, introspect, generate, db pull & db push) except the  new PrismaClient() which generate the issue mentioned above

my work around was that i inserted the db url into the schema.prisma file @url = "" 
and it worked, but i want to know, what is the issue? why can't the database url be read from the .env file?

Thank you

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
